lib/models.ts

```
@@ -0,0 +1,146 @@
+}
+
+/**
+ * Initialize and return an AI model instance based on the provided model ID
+ *
+ * @param modelId - The ID of the model to initialize
+ * @param useWebSearch - Whether to enable web search capabilities
+ * @returns The initialized model and its display name
+ */
+export function getInitializedModel(
+  modelId: string,
+  useWebSearch: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)